RG_CHANGELOG.rst

```diff
@@ -16,3 +16,5 @@ Added:
 - add CI checkers: tests, linters checkers
 - add configs and tests
 - add documentation
+
+* no-missing-end-of-source-newline rule to stylelint
```

Comment:
[nit] Let's fix file structure:
```
Added:
======
* no-missing-end-of-source-newline rule to stylelint (RGInt-303)
* initialize project structure and add default configs (RGInt-303)

  - add CI checkers: tests, linters checkers
  - add configs and tests
  - add documentation
```
It will render:
![image](https://github.com/user-attachments/assets/472ea2b9-f10c-426f-8e08-321904391760)

Reply:
Changed